images/no-bookmark.svg

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
uh what is this? i do not really understand it 😅

Reply:
i think this is from the bookmark svg 😅
I don't understand it too 🤷🏻